src/XMakeBuildEngine/Evaluation/ProjectParser.cs

```diff
@@ -194,32 +194,23 @@ private void ParseProjectElement(XmlElementWithLocation element)
                 // TODO: multiples
                 //  TODO: Don't null ref if MSBuildSDKsPath isn't defined
 
-                // TODO: paths should just be Sdk.props/targets; Sdk-aware imports should do the rest of the path.
-                var initialImportPath = Path.Combine(Environment.GetEnvironmentVariable("MSBuildSDKsPath"),
-                    element.GetAttribute("Sdk"), "Sdk", "Sdk.props");
-                var finalImportPath = Path.Combine(Environment.GetEnvironmentVariable("MSBuildSDKsPath"),
-                    element.GetAttribute("Sdk"), "Sdk", "Sdk.targets");
-
-                if (File.Exists(initialImportPath))
+                //  TODO: Add file existence conditions to the imports
+                //if (File.Exists(initialImportPath))
```

Comment:
The TODO I added to `GetCurrentDirectoryForConditionEvaluation` should help with this.  Then an `Exists` call in a condition would evaluate paths relative to the resolved Sdk folder.  However, it should be an error to try to refer to an Sdk that doesn't exist in the Sdk attribute on the project (because we want a good error if the Sdk is misspelled or not installed), which simply adding `Exists` conditions to the implicit imports won't help with.

I feel like this is one of the things that we could handle better if we handled the Sdk imports in the evaluator rather than adding implicit elements for them in the `ProjectParser`.